src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        action="#"
+        method="get"
+        class="form-searchbar searchbar"
+        data-qa="big"
+      >
+        <button type="submit" class="searchbar__button">
```

Comment:
Because we use `data-qa="keypress"` you will create behavior when you press on `Enter` button. So, you don't need this button.